_episodes/37.md

```diff
@@ -0,0 +1,261 @@
+for healthy projects.
+
+As such, the Trino Software Foundation (TSF) recognizes that Trino will only be
+as successful as the health of the community that builds, tests, uses, and 
+shares it. The activities around building a technical community falls in between
+engineering, marketing, and customer enablement. A common name that encompasses
+the individuals that work in this space is developer relations, DevRel for
+short. The TSF DevREl team has recently grown from the founding Developer
+Advocate, Brian, to the many folks on the call today. The mission around this
+team is as follows:
+
+> Grow all aspects of the Trino project, and the Trino community to empower
+> current and future members of the community.
+
+This episode is an attempt to bring some transparency in our goals with this 
+team, the , and how that affects all of the members in this community.
+
+### Improving the Onboarding and Getting Started Pages (Brian + Monica)
+
+We don't really have a seamless onboarding experience for new users. Many 
+members have asked questions on where to get started. One logical place people
+tend to go to when browsing on the front page of the Trino site is the 
+[getting started tab](https://trino.io/download.html), which is ironically 
+still on the `trino.io/download.html` page. When you open this page, you are
+brought to a page primarily containing the latest binary downloads, some
+community links, and some reading material to books and other resources.
+
+The main thing you don't really see is much getting started material. A lot of
+the material is intermediate level at best. There is not much beginner level
+guides to offer the self-service onboarding many are looking for when they just
+want to play around without having to bother or wait for anyone to respond. As
+it stands today, there is some work that Brian and Monica have started to create
+in this area to make the onbaording simpler. 
+
+A very common self-service getting started material is the 
+[trino-getting-started](https://github.com/bitsondatadev/trino-getting-started)
+repo that Brian created to house some demonstrations he does on this broadcast
+to show off some new feature or connector capabilities. This has been a good
+way to offer folks a simple environment to get them started, but the only way
+to find this repository is to ask someone first. It would be ideal to showcase
+getting started materials as part of the default experience of learning about
+Trino.
+
+Monica is working now on building up some demos using SaaS products like
+Starburst Galaxy as another method of using Trino without needing to install
+Docker among having to use any of your hardware to run through some examples.
+These options are typically more UI driven and much more approachable by
+members of the community that aren't engineers or administrators that simply
+want to try Trino without much setup.
+
+### Release Process (Cole)
+
+#### Filling out PR
```

Comment:
```suggestion
#### Filling out PR

We've got a handy PR template that exists for all contributors to use when they've submitting a pull request to Trino. Most of it is simple and self-explanatory - we ask you to describe what's happening, where the change is happening, and what type of change it is. These are for the sake of the reviewers, giving them some important context so they understand what's going on when they review the code. For simpler changes, it's not usually necessary to go into a ton of detail here, but it's usually nice to give a little summary if you think it's appropriate.

The next steps are what really matter for every single PR that's going to be merged - the documentation and release notes for a change. These are about communicating to our users. Documentation refers to Trino docs, not code comments - if Trino users will need to be told how to use the feature you're changing because of how you're changing it, that means we need to have documentation for it. The PR template gives the options for how to go about this, but it's incredibly helpful to have this filled out. Similarly, we ask whether or not release notes are necessary for the change, and what release notes you would propose for your change. Generally speaking, if it needs to be documented, it almost always should have a release note. Even if it isn't documented, a release note might be a good idea - things like performance improvements don't require our users to change how they use Trino, but they won't mind knowing that something has gotten faster/better! The release process involves heavy editing of release notes, so it's ok for the suggested note to be imperfect.
```